### DISCUSSION TITLE
Aiuto nuove implementazioni

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#783] [Richieste di aiuto]
Buonasera,ho provato a modificare un po' l'aspetto riguardante lato quadro voti docente e devo dire che il tutto funziona benissimo  ,inoltre invece della tipologia di valutazione scritto/grafica ho staccato creando due tipologie :una scritta e una pratica e ho aggiunto un'altra chiamata altro/unico e devo dire che il tutto sembra andare bene ,almeno apparentemente ,ma quando mi reco poi ad inserire una valutazione di qualsiasi tipo ,mi ritrovo il seguente messaggio non appena clicco sulla tipologia di valutazione da inserire :"Devi prima scegliere la classe o la materia per poter visualizzare questa pagina." ,nonostante io abbia selezionato la classe ,questa cosa non mi permette di inserire valutazioni di alcun tipo .Allego i file modificati 
[voti_quadro.html.twig prova.zip](https://github.com/user-attachments/files/17722239/voti_quadro.html.twig.prova.zip)
[VotiController (3).php.zip](https://github.com/user-attachments/files/17722240/VotiController.3.php.zip)

**Comment by sempliciniluigi89** [comment]
qualcuno potrebbe aiutarmi a capire dove sbaglio?

**Comment by trinko** [comment]
Non ho tempo per analizzare i dettagli per capire l'errore che indichi, ma non vedo le modifiche al database.
Se aggiungi un nuovo dato, ci deve essere un nuovo campo nell'entità **src/Entity/Valutazione.php** e, di conseguenza, nella corrispondente tabella del database.
Anche il nuovo tipo deve essere tra quelli ammessi nella variabile **tipo**, sempre nello stesso file.
